package.json

```diff
@@ -37,12 +37,16 @@
   "scripts": {
     "watch": "nodemon --watch scss/ --ext scss --exec \"npm run css-main\"",
     "build": "npm-run-all css-compile-main css-prefix-main css-minify-main copy-tinymce-ui-css",
-    "css-main": "npm-run-all css-compile-main css-prefix-main css-minify-main",
+    "build-toolbar": "npm-run-all css-compile-toolbar css-prefix-toolbar css-minify-toolbar",
+    "css-main": "npm-run-all css-compile-main css-minify-main",
     "css-compile-main": "sass --load-path=node_modules --style expanded --source-map --embed-sources --no-error-css scss/barceloneta.scss:plonetheme/barceloneta/theme/css/barceloneta.css",
     "css-prefix-main": "postcss --config postcss.config.js --replace \"plonetheme/barceloneta/theme/css/*.css\" \"!plonetheme/barceloneta/theme/css/*.min.css\"",
     "css-minify-main": "cleancss -O1 --format breakWith=lf --with-rebase --source-map --source-map-inline-sources --output plonetheme/barceloneta/theme/css/barceloneta.min.css plonetheme/barceloneta/theme/css/barceloneta.css",
     "css-lint": "stylelint \"scss/**/*.scss\" --cache --cache-location .cache/.stylelintcache",
-    "copy-tinymce-ui-css": "cp \"node_modules/tinymce/skins/ui/oxide/content.css\" \"plonetheme/barceloneta/theme/tinymce/tinymce-ui-content.css\""
+    "copy-tinymce-ui-css": "cp \"node_modules/tinymce/skins/ui/oxide/content.css\" \"plonetheme/barceloneta/theme/tinymce/tinymce-ui-content.css\"",
+    "css-compile-toolbar": "sass --load-path=node_modules --style expanded --source-map --embed-sources --no-error-css scss/edittoolbar.scss:plonetheme/barceloneta/theme/css/edittoolbar.css",
```

Comment:
Does this work for you? there is no `scss/edittoolbar.scss` in master branch. Is there something missing here?

Reply:
that's right, I have just pushed it.My fault.